packages/react/src/components/Embed/Embed.tsx

```diff
@@ -0,0 +1,145 @@
+import * as PropTypes from 'prop-types'
+import * as React from 'react'
+
+import {
+  createShorthandFactory,
+  UIComponent,
+  UIComponentProps,
+  commonPropTypes,
+  applyAccessibilityKeyHandlers,
+  customPropTypes,
+} from '../../lib'
+import { embedBehavior } from '../../lib/accessibility'
+import { Accessibility, AccessibilityActionHandlers } from '../../lib/accessibility/types'
+import Image from '../Image/Image'
+import Video from '../Video/Video'
+import Box from '../Box/Box'
+import * as _ from 'lodash'
+
+import { ReactProps, ShorthandValue } from '../../types'
+
+export interface EmbedProps extends UIComponentProps {
+  /**
+   * Accessibility behavior if overridden by the user.
+   * @default embedBehavior
+   * */
+  accessibility?: Accessibility
+
+  /** Video source URL. */
+  src?: string
```

Comment:
We can remove `src` as we will have correctly working shorthand, https://github.com/stardust-ui/react/pull/1108/files#r272260321